docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
     build: ./deploy-service/teletraanservice
     ports:
       - "8080:8080"
-      - "8081:8081"
+      - "8082:8082"
```

Comment:
Avoid port conflict on devapp

Reply:
what is the conflict?

Reply:
When I ran on devapp, port 8081 is already used by other containers. Not a big deal, but changing the port will make it run the 1st time without us manually changing the port.